test/e2e/fixtures.go

```diff
@@ -109,6 +109,19 @@ func setupTestWithIPFIXCollector(tb testing.TB) (*TestData, error) {
 	return data, nil
 }
 
+func createDirForCoverage(covDir string) error {
+	// a filepath-friendly timestamp format.
+	// remove directory if it already exists. This ensures that we start with an empty
+	// directory. Given that we append a timestamp at the end of the path it is very unlikely to
+	// happen.
```

Comment:
does that comment apply there? I don't see a timestamp being added to the directory name

Reply:
let me remove the comment here. We used a specified directory to store the coverage files.